actiontext/app/helpers/action_text/tag_helper.rb

```diff
@@ -50,7 +50,16 @@ def render
       options = @options.stringify_keys
       add_default_name_and_id(options)
       options["input"] ||= dom_id(object, [options["id"], :trix_input].compact.join("_")) if object
-      @template_object.rich_text_area_tag(options.delete("name"), options.fetch("value") { value }, options.except("value"))
+      html_tag = @template_object.rich_text_area_tag(options.delete("name"), options.fetch("value") { value }, options.except("value"))
+      if object_has_errors?
+        @template_object.instance_exec(html_tag, self, &ActionView::Base.field_error_proc)
+      else
+        html_tag
+      end
```

Comment:
Would it be possible to reuse the implementation of `ActiveModelInstanceTag`?
https://github.com/rails/rails/blob/27d24e9caa1942fb29f87bc14f66b5a0aaf53773/actionview/lib/action_view/helpers/active_model_helper.rb#L28

```suggestion
      error_wrapping(html_tag)
```
If not, can we maybe we include that helper?